README.md

```diff
@@ -1,175 +1,404 @@
-# postgres-mcp: PostgreSQL Tuning and Analysis Tool
+# Postgres Pro | MCP Server
 
-postgres-mcp is a Model Context Protocol (MCP) server that helps you analyze, optimize, and monitor your PostgreSQL databases. It allows AI assistants like Claude to analyze query performance, recommend indexes, and perform health checks on your database.
+[ [Quick Start](#quick-start) | [Intro Blog Post](https://www.crystaldba.ai/blog) | [Discord Server](https://discord.gg/4BEHC7ZM) ]
```

Comment:
We should add version information (current release, compatibility matrix with Postgres versions, etc).